sw/airborne/firmwares/rotorcraft/guidance/guidance_v.c

```diff
@@ -232,6 +232,10 @@ void guidance_v_run(bool in_flight)
     case GUIDANCE_V_MODE_RC_DIRECT:
       guidance_v.z_sp = stateGetPositionNed_i()->z; // for display only
       stabilization_cmd[COMMAND_THRUST] = guidance_v.rc_delta_t;
+
+#ifdef SetCommandsFromRC
```

Comment:
This doesn't make sense to me, first this is set
```stabilization_cmd[COMMAND_THRUST]```
And then it is overwritten. I also don't understand why this would be part of vertical guidance. I think it would be better if there was an explicit check if some thrust_x command is available, and otherwise (explicitly) take a certain RC channel. In my eyes that would be more transparent than a macro that is quite hard to find and understand.

Reply:
Ok, in ```guidance_h``` as requested and a clear ```RADIO_CONTROL_THRUST_X``` instead of a hidden define.